### PR TITLE
[UI] Fix minor styling bugs

### PR DESCRIPTION
**Pull request by Etaash-mathamsetty**
After:
image
(my mouse was over the play button when I took this screenshot)
image

Before:
image
image

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [x] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

File: src/frontend/components/UI/Anticheat/index.scss [hunk 1]
```diff
@@ -6,6 +6,7 @@
   border-radius: 20px;
   background-color: var(--current-background);
   margin-bottom: var(--space-lg);
+  margin-top: var(--space-lg);
```
Comment (flavioislima) [inline src/frontend/components/UI/Anticheat/index.scss:9]: can you check if it is affecting the install dialog as well? since it uses the same component?

**Review by flavioislima — APPROVED** [review]
Looks good, but would be good to check the install dialog as well

File: src/frontend/components/UI/Anticheat/index.scss (hunk 1 above)
Comment (Etaash-mathamsetty) [reply src/frontend/components/UI/Anticheat/index.scss:9]: install dialog is looking fine

**Comment by flavioislima** [comment]
tested here, looks fine

**Pull request merged**